scripts/mail-all-activity.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+'''Mail people to check their activity based on YAML dump of Amy database.'''
+
+import sys
+import yaml
+
+
+USAGE = 'Usage: mail-all-activity [-r|--real] [filename]'
+
+BODY = '''We would like to bring our records up to date, and would be grateful
+if you could check the information below and send us corrections or additions.
```

Comment:
I'd change the message a little. I'd add "what to do if my skills, email, or tasks don't match", and "what if everything's okay".